package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@safetyculture/sharepoint",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
Looks like this was incremented by hand - there's actually a process to do the release. I'll walk through it next week.

(Otherwise, change below looks good)